code/mob/living/critter/flock/flockdrone.dm

```diff
@@ -487,14 +475,12 @@
 				var/turf/simulated/wall/auto/feather/wall = src.loc
 				if (wall.broken)
 					return ..()
+				if (!wall.on)
+					wall.on()
```

Comment:
This looks spooky, but testing with medass shows they don't conflict since DM doesn't treat methods as properties

Reply:
based non-first-class-functions